lasertagger/run_lasertagger.py

```diff
@@ -224,6 +230,18 @@ def main(_):
   else:
     num_train_steps, num_warmup_steps = None, None
 
+  
+  if FLAGS.verb_loss_weight < 0:
+        raise ValueError("the weight of verb loss should be >= 0")
+  
+  with open(os.path.expanduser(FLAGS.label_map_file)) as f:
+    lines = f.readlines()
```

Comment:
the results of f.readlines() may contain '\n' in each item, you may need to call strip():
https://stackoverflow.com/questions/3277503/how-to-read-a-file-line-by-line-into-a-list

Reply:
Thanks a lot for your suggestions! Here, I am searching for whether the line contains "DELETE", so the newline or space characters should not matter, but I added the strip() just in case. Thanks a lot!

Reply:
Got it thanks.